torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1850,6 +1850,85 @@ def sample_inputs_stack(op_info, device, dtype, requires_grad, **kwargs):
         for dim in range(-1, len(shape) - 1):
             yield SampleInput(tensors, args=(dim,))
 
+def error_inputs_cat(op_info, device, **kwargs):
+    # error inputs for unsupported operation
+    x = torch.rand((1, 3), device=device).expand((6, 3))
+    y = torch.rand((3, 3), device=device)
+    yield ErrorInput(SampleInput((y, y), kwargs={'out': x}),
+                     error_regex='unsupported operation')
+
+    # error inputs for empty tensors
+    yield ErrorInput(SampleInput([], kwargs={'dim': 1}),
+                     error_regex='non-empty list of Tensors')
+
+    # error inputs for different sizes
+    dtype = torch.float32
+    x = torch.randn((4, 3, 32, 32), dtype=dtype, device=device)
```

Comment:
Instead of `randn` use `make_tensor` (which this can partial) like other sample/error input functions

Reply:
Yep! It was fun. Updated it. Thanks, @mruberry!